word-addin/src/CyBy/Draw/Word.idr

```diff
@@ -104,7 +127,8 @@ exportImage svg mol =
   wordRun $ \c => do
     debug "Begin of function `exportImage`"
     -- replace the selected svg (or the first in the selection)
-    -- with the updated structure if an svg is selected
+    -- with the updated structure and adjust the size accordingly
+    -- if an svg is selected
```

Comment:
It looks like currently we do not adjust the size of a freshly inserted image but only adjust sizes when we update an existing image. We should have consistent sizes for all images.